Resolve #498 (#521)
worldbank/dime-data-handbook: chapters/2-collaboration.tex

% ----------------------------------------------------------------------------------------------
\subsection{Setting up a productive code environment}

> Taking time early in your project to choose a programming language to work on and

Reviewer: ```suggestion
Taking time early in your project to choose a programming language to work in and
```

Combining these two types of collaboration tools
almost always creates undesired functionalities.

\subsection{Connecting tasks through files}
Each task in the research workflow has specific inputs and outputs
which feed into one another.
It is common, particularly when different tasks are performed by different people inside a team,
for incompatibilities to be created.
For example, if the Principal Investigators are writing a paper using {\LaTeX},
exporting tables from statistical software into a \texttt{.csv} format will break the workflow.
Therefore, it's important to agree with your team on what tools will be used for what tasks, 
and where inputs and outputs will be stored, before you start creating them. 
Take into account ease of use for different team members, 
and keep in mind that learning how to use a new tool may require some time investment upfront that will be paid off as your project advances.

Knowing how code outputs will be used will help you decide the best format to export them. 

> You can typically use the same command to save figures into different formats,

Reviewer: ```suggestion
You can typically use the same software to save figures into various formats,
```

such as \texttt{.eps}, \texttt{.png}, \texttt{.pdf} or \texttt{.jpg}. 
However, the decision between using Office Suite software 

> such as Word and Power Point versus {\LaTeX} and other plain text formats 

Reviewer: ```suggestion
such as Word and PowerPoint versus {\LaTeX} and other plain text formats 
```

may influence how you write your code,

> as this choice often implicates in the use of a particular command. 

Reviewer: ```suggestion
as this choice often implicates in the use of a particular format. 
```

> This decision will also affect the version control systems that your team can use,

Reviewer: ```suggestion
This decision will also affect the version control systems that your team can use.
```
